_posts/programme/2020-08-19-kickoff.md

```diff
@@ -14,42 +14,26 @@ registration_url: https://docs.google.com/forms/d/e/1FAIpQLSetlGKtfCxxaAjtHEGxaQ
 meeting_url: https://zoom.us/j/95982897830
 last_modified_at: 2020-09-02
 ---
-To kick our series off, we’re delighted to announce that the SORSE Launch
-Event will take place 13:00-15:00 UTC on Wednesday 2nd September. We’re
-pleased to be welcoming two keynote speakers at this event:
+To kick off our series off, we have run the SORSE Launch
+Event on Wednesday 2nd September. We had
+two keynote speakers at this event:
 
 - Dr Mariann Hardey, Associate Professor, Durham University: _Switching off the label "women in tech"_
 - Dr Kari Jordan, Executive Director, The Carpentries: _I wanna dance with somebody_
 
-<div>
-    {% include registration-button.html %}
-    {% include zoom-button.html %}
-    {% assign time = page.time %}
-    {% include add-to-calendar-button.html %}
-</div>
-
-The session will begin with a short welcome and introduction to SORSE followed
-by two 30 minute keynote talks. Dr Mariann Hardey will speak on the subject of
-women in tech and Dr Kari Jordan will speak about the Carpentries (both
-recorded). Further details and abstracts can be seen below. The talks will be
+The session began with a short welcome and introduction to SORSE followed
+by two 30 minute keynote talks. Dr Mariann Hardey spoke on the subject of
+women in tech and Dr Kari Jordan spoke about the Carpentries. The talks were
 followed by time for questions and then an opportunity for further discussion
 and networking in breakout rooms.
 
-## Joining instructions
-The event will take place as a Zoom Webinar and will switch to a regular Zoom session for discussion and networking in breakout groups. Connection details
-will be sent to all registered participants in advance of the event.
-
-If you haven't registered, you can still join by clicking the _Join_ button
-above.
+<i class="fab fa-fw fa-youtube"></i> A recording of this event is [available on YouTube](https://www.youtube.com/watch?v=2HOznlOCd1w).
```

Comment:
What about also adding the video directly into the site? Should be easy through the [youtube helper](https://mmistakes.github.io/minimal-mistakes/docs/helpers/#youtube) provided by our theme.

Reply:
Sure!